### ISSUE TITLE
how do i set tablacus to show hidden/system files. 

### ISSUE THREAD

**Issue by astro46** [tablacus/TablacusExplorer#631]
system and hidden files aren't showing.  I can see them in windows explorer.  
I can't find a setting for this. 
in options>tree (what a strange place to put these settings), i have checked 'include hidden files", and unchecked 'hide protected operating system files'.

win10,x64.  current version of tablacus.

**Comment by astro46** [comment]
never mind:  i found it. under options>list   show all files.   how come the settings mentioned above didn't do this?

**Comment by ccfs** [comment]
"Options > Tree" are the settings of "folder tree".

"Options > List" are the settings of "file list". "Show all files" controls whether or not hidden (and system) items will show. However TE does NOT have a "Hide protected operating system files" setting for file lists. It follows "Folder options" of Windows.

**Comment by astro46** [comment]
I don't know about the differences in settings between folder trees and file lists.  but i can say that tablacus definitely did not follow the "folder options" of windows.   I did not see the hidden and system files in tablacus, but did when i switched to using Win Explorer.   Until i changed the setting in tablacus.

options>tree>"hide protected operating system file (recommended) was checked.

and options>list> "show all files (including hidden items) was also checked.
i unchecked both.

**Comment by ccfs** [comment]
This is my test result:
![mx](https://user-images.githubusercontent.com/34089121/205538605-bb33cc94-de3c-4730-99d1-babddf07a720.png)
"Folder options" of Windows DOES affect file lists in TE (green Yes).
p.s. the red Yes is probably a TE problem.

**Comment by astro46** [comment]
Wow.  that is pretty elaborate. I guess I put together charts that show similar info in cases like this, though not as clean and moveable as yours.  impressive. 
However, I don't know if this chart confirms or contradicts what i said previously. I do know that I saw hidden folders and files in win explorer, but not in TE, until after I changed the the three settings.  Though i never tried just changing options>list>'show all files' settings while leaving the first 2 mentioned, turned off.
